doc/contribute/guidelines.rst

```diff
@@ -554,6 +554,31 @@ exceptions:
 * Avoid using binary literals (constants starting with ``0b``).
 * Avoid using non-ASCII symbols in code, unless it significantly improves
   clarity, avoid emojis in any case.
+* Use proper capitalization of nouns in code comments (e.g. ``UART`` and not
+  ``uart`` or ``CMake`` ``cmake``).
+
+Beyond C code, the following coding style rules apply to other types of files:
```

Comment:
cool though we should really have some compliance check validation for these to have any hope of being consistent, but hey guess this is the first step

insert: I'm a sign not a cop meme